changelog/upgrade-notes.md

```diff
@@ -2,6 +2,12 @@
 
 ### Next version
 
+#### Deprecation of `ol/AssertionError` and error codes
+
+Future versions will no longer throw `ol/AssertionError` with an error `code`. Instead, they will throw `Error` with just the error message.
+
+### v7.0.0
+
```

Comment:
This file looks out of date

Reply:
Indeed. I just rebased the pull request.